azure-mgmt-sql/src/main/java/com/microsoft/azure/management/sql/ReplicationLink.java

```diff
@@ -77,10 +77,19 @@
      */
     ReplicationState replicationState();
 
-
     /**
      * Deletes the replication link.
      */
     void delete();
+
+    /**
+     * Failover the Azure SQL Database Replication Link with the given id.
```

Comment:
javadocs decription should be written in 3rd person declarative mood, i.e. "Fails over ..." not "Failover, and "Forces ..." and not "Force...". This issue exists in many places.